### PR TITLE
Accessible win loss

### PR DESCRIPTION
**Pull request by GreenAsJade**
Fixes win/loss being red/green which colour blind people find hard.

## Proposed Changes

~~Make win == yellow and loss == blue~~

Make win == blue and loss ~= red

### CODE REVIEW

**Comment by BHydden** [comment]
could I request the opposite? win = blue and loss = yellow?

yellow is closer to red, and are both warm colours, so they associate well.
and blue is cool like green, and also has positive connections such as an AI "blue move" being optimal.

**Comment by BHydden** [comment]
Actually, blue/yellow is another type of colorblindness along with red/green... so maybe blue/red would be better?

keep loss == red and change win from green to blue?

**Comment by GreenAsJade** [comment]
OK.

This seems like a well researched suggestion.

I do wish that you had made this suggestion when I asked, in the forum, rather than after I went to the trouble of implementing the suggestion that was given in response to that question.   :S  

Curious that the colourblind person themselves didn't come up with this answer.

**Comment by GreenAsJade** [comment]
... and happily, the tweaks I'd had to do going from dark->yellow/blue held up well with blue/vermillion, so that was more painless than I exptected :)

**Comment by GreenAsJade** [comment]
Screen Shot 2021-02-13 at 8 47 00 pm

Screen Shot 2021-02-13 at 8 47 07 pm

**Comment by BHydden** [comment]
Sorry, I haven't been as up to date with all the forum threads that I would have liked. Glad it wasn't too painless (and actually it looks kinda cool! I may even like those colours over our current theme haha)

**Comment by GreenAsJade** [comment]
Yeah no worries  TBH I had hoped to find a chart like the one you did, and I agree that this looks pretty reasonable - rather better than the yellow/blue - so it's all win.

**Comment by BHydden** [comment]
I bookmarked it from [the trophy conversation](https://forums.online-go.com/t/31581/12)....

**Comment by GreenAsJade** [comment]
@anoek  Our colourblind guineapig reckons [this does the job](https://forums.online-go.com/t/colors-of-filled-online-status-circles-difficult-to-distinguish-for-colorblinds/24240/45?u=eugene).

Needs/brings #1391

**Comment by flovo** [comment]
I just found one can use chromes development tools to emulate various vision deficites: https://chiamakaikeanyi.dev/using-chrome-devtools-to-emulate-vision-deficiencies/
